docs/man/docker-create.1.md

```diff
@@ -110,6 +111,18 @@ IMAGE [COMMAND] [ARG...]
 **-m**, **--memory**=""
    Memory limit (format: <number><optional unit>, where unit = b, k, m or g)
 
+   Allows you to constrain the memory available to a container. If the host
+supports swap memory, then the -m memory setting can be larger than physical
+RAM. If a limit of 0 is specified, the container's memory is not limited. The
+actual limit may be rounded up to a multiple of the operating system's page
+size(the value would be very large, near the largest unsigned long long).
```

Comment:
Missing space after "size". Suggest "close to" instead of "near". Maybe also clarify a little by specifying "largest unsigned long long integer."? I'm not sure how familiar the average user is with these concepts.